database/schema/create_schema.sql

```diff
@@ -587,6 +588,7 @@ CREATE TRIGGER system_advisories_set_first_reported BEFORE INSERT ON system_advi
   FOR EACH ROW EXECUTE PROCEDURE set_first_reported();
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON system_advisories TO evaluator;
+GRANT SELECT, INSERT, UPDATE, DELETE ON system_advisories TO listener;
```

Comment:
We'll have to remove this permissions once we separate evaluator from listener.

Reply:
Yes, but it wouldn't be too difficult :smile: